.github/workflows/log-pr-reviewer.yml

```
@@ -0,0 +1,39 @@
+      CLIENT_ID: ${{ secrets.CLIENT_ID }}
+      CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+      TOKEN_URL: ${{ vars.TOKEN_URL }}
+
+    steps:
+      - name: Get OAuth2 Access Token
+        id: get_token
+        run: |
+          response=$(curl --silent --request POST "$TOKEN_URL" \
+            --header "Content-Type: application/x-www-form-urlencoded" \
+            --data "grant_type=client_credentials&client_id=$CLIENT_ID&client_secret=$CLIENT_SECRET")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)